include/dt-bindings/board/am572x-bbai-pins.h

```diff
@@ -106,4 +106,610 @@
 #define P9_42A(mode) DRA7XX_CORE_IOPAD(0x36E4, mode) /* E14: P9.42a: mcasp1_axr12 */
 #define P9_42B(mode) DRA7XX_CORE_IOPAD(0x359C, mode) /* C2: P9.42b: vin2a_d13 */
 
+
+/********************************/
+/* BBAI P8 Header pinmux macros */
+/********************************/
+/* P8_01               		GND */
+/* P8_02               		GND */
+
+/* P8_03  (ball AB8) gpio1_24 */
+#define P8_03_DEFAULT       P8_03( PIN_OUTPUT_PULLDOWN | INPUT_EN | MUX_MODE14)	    /* mmc3_dat6.gpio1_24 */
```

Comment:
Are tabs still off in your editor? Things don't seem to line up for me.

Reply:
Are these defines somehow better than the definitions that could be referenced as symbols before? Is the point to save space in the .dtb? Please include your reasoning for the changes.

Reply:
you are seeing an outdated version of the file.